helm/templates/_helpers.tpl

```diff
@@ -88,7 +88,7 @@ Return the Postgresql user
 */}}
 {{- define "odoo.databaseUser" -}}
 {{- if .Values.postgresql.enabled }}
-    {{- printf "%s" .Values.postgresql.postgresqlUsername -}}
+    {{- printf "%s" .Values.postgresql.auth.username -}}
```

Comment:
Hey @jeremi, I see you've changed the variable names here. Was there any particular reason for the change? 

And, normally we expect the PR to contain changes in the values.yml file for the corresponding variable name updates.

Reply:
yes, they were changed in the bitnami helm, so this is why it broke this helm:
https://github.com/bitnami/charts/commit/0bcac356090f8f0eb5d2ad53a7ccfeeaa633a485#diff-cd2915619fd284ca81ebd3bf58856844260eb608bb270d7f07a58621bbdc4cbbL228